source/Patches/CrewmateRoles/MedicMod/DeadBody.cs

```diff
@@ -33,39 +69,36 @@ public static string ParseBodyReport(BodyReport br)
                 return
                     $"Body Report: The killer appears to be {br.Killer.Data.PlayerName}! (Killed {Math.Round(br.KillAge / 1000)}s ago)";
 
-            var colors = new Dictionary<int, string>
-            {
-                {0, "darker"},// red
-                {1, "darker"},// blue
-                {2, "darker"},// green
-                {3, "lighter"},// pink
-                {4, "lighter"},// orange
-                {5, "lighter"},// yellow
-                {6, "darker"},// black
-                {7, "lighter"},// white
-                {8, "darker"},// purple
-                {9, "darker"},// brown
-                {10, "lighter"},// cyan
-                {11, "lighter"},// lime
-                {12, "darker"},// maroon
-                {13, "lighter"},// rose
-                {14, "lighter"},// banana
-                {15, "lighter"},// gray
-                {16, "darker"},// tan
-                {17, "lighter"},// coral
-                {18, "darker"},// watermelon
-                {19, "darker"},// chocolate
-                {20, "lighter"},// sky blue
-                {21, "darker"},// beige
-                {22, "lighter"},// hot pink
-                {23, "lighter"},// turquoise
-                {24, "lighter"},// lilac
-                {25, "lighter"},// rainbow
-                {26, "lighter"},// azure
-            };
-            var typeOfColor = colors[br.Killer.Data.ColorId];
+            //Get the colorType of the Killer
+            var typeOfColor = ColorType[br.Killer.Data.ColorId];
             return
                 $"Body Report: The killer appears to be a {typeOfColor} color. (Killed {Math.Round(br.KillAge / 1000)}s ago)";
         }
     }
+
+    //Update meeting to clorize the darker color name in black
+    [HarmonyPriority(Priority.Last)]
+    [HarmonyPatch(typeof(HudManager), nameof(HudManager.Update))]
+    public static class HudManager_Update
+    {
+        private static void PostFix(MeetingHud __instance)
+        {
+            var localPlayer = PlayerControl.LocalPlayer;
+            if (MeetingHud.Instance == null) return;
+            if (localPlayer.Data.IsDead) return;
+            if (!localPlayer.Is(RoleEnum.Medic)) return;
+
+            foreach (var playerState in __instance.playerStates)
+            {
+                var player = PlayerControl.AllPlayerControls.ToArray()
+                    .FirstOrDefault(x => x.PlayerId == playerState.TargetPlayerId);
+
+                if (localPlayer != player && player != null && player.Data != null)
+                {
+                    if (BodyReport.ColorType[player.Data.ColorId] == "darker")
```

Comment:
Do we also need to consider camouflaged meetings?

Reply:
Mmh may be, i test for seer, role criteria and camouflage meetings, and i fix if needed

Reply:
So I tested, and since I moved the function out of Role.cs and put it on the MedicMod folder side, the names no longer turn black, I tried in vain, playing with the HarmonyPriority without success.
Otherwise for camouflaged meetings there is no risk of having a problem, since the names do not appear so no black colors either.

Reply:
Are you sure you set priority correctly? I know for a fact it works.

Reply:
I set first for the Medic méthode and high for Role.cs

Reply:
Well yeah, if Medic happens first then Role will overwrite it